web/src/compose_validate.ts

```diff
@@ -384,15 +382,13 @@ export function warn_if_mentioning_unsubscribed_group(
     }
 
     const $banner_container = compose_banner.get_compose_banner_container($textarea);
-    if (
-        $banner_container.find(
-            `.${CSS.escape(compose_banner.CLASSNAMES.group_entirely_not_subscribed)}`,
```

Comment:
You still need to be looking inside this area. Otherwise, if there were some random unrelated reason a banner might mention a group, this logic will handle it wrong.